src/main/java/datawave/util/time/DateHelper.java

```diff
@@ -156,10 +203,15 @@ public static Date parseHour(String date) {
      */
     private static Date lenientParseHelper(String date, DateTimeFormatter parser, String formatStr, boolean hasTime) {
         String lenientDate = convertToLenient(date, formatStr);
-        if (hasTime) {
-            return Date.from(ZonedDateTime.parse(lenientDate, parser).toInstant());
-        } else {
-            return Date.from(LocalDate.parse(lenientDate, parser).atStartOfDay(parser.getZone()).toInstant());
+        try {
+            if (hasTime) {
+                return Date.from(ZonedDateTime.parse(lenientDate, parser).toInstant());
+            } else {
+                return Date.from(LocalDate.parse(lenientDate, parser).atStartOfDay(parser.getZone()).toInstant());
+            }
+        } catch (DateTimeParseException e) {
+            log.warn(e.getMessage());
```

Comment:
Logging and then throwing will typically lead to multiple copies of this exception being logged. It's better to log and handle it OR just throw it and let whoever catches log it. In this case, just throwing it makes sense to me.

Reply:
Done